src/components/attendee/MiniProfile/components/MiniProfileModalLinks.tsx

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { ProfileLink } from "types/User";
+
+import { getProfileModalLinkIcon } from "utils/profileModalLinkUtilities";
+
+import styles from "./MiniProfileModalLinks.module.scss";
+
+interface MiniProfileModalLinksProps {
+  profileLinks?: ProfileLink[];
+}
+
+export const MiniProfileModalLinks: React.FC<MiniProfileModalLinksProps> = ({
+  profileLinks,
+}) => {
+  const renderedProfileModalLinks = useMemo(
+    () =>
+      profileLinks?.map((link: ProfileLink) => {
+        const linkIcon = getProfileModalLinkIcon(link.url);
+
+        return (
+          <a key={link.url} href={link.url} target="_blank" rel="noreferrer">
```

Comment:
This might also be a security risk, hard-coding `noreferrer` and not having `noopener`. It would be better if all these cases refer to the same source:

```ts
export const externalUrlAdditionalProps = {
  target: "_blank",
  rel: "noopener noreferrer",
};
```

Reply:
oh yes, thanks! Fixed: https://github.com/sparkletown/sparkle/pull/2953/commits/a52527d7fce74b5f15447723260cc7715c8614b3